app/assets/stylesheets/styles/components/guides/components/_step_1.css.scss

```diff
@@ -0,0 +1,30 @@
+/* Step 1 */
+.getting-started {
+
+  .crop-name-and-description {
+    margin-bottom: 1.5rem;
+  }
+
+  .crop-name {
+    font-size: 1.8rem;
+    /* account for cancel button */
+
+    .common-name {
+      padding-right: 1.5rem;
+    }
+
+    .binominal-name {
+      color: #666;
```

Comment:
Color literals like `#666` should only be used in variable declarations; they should be referred to via variable everywhere else.